docs/ingest-data/sqs-files.md

```diff
@@ -0,0 +1,235 @@
+    events    = ["s3:ObjectCreated:*"]
+  }
+}
+```
+
+The source needs to have access to both the notification queue and the source
+bucket. The following policy document contains the minimum permissions required
+by the source:
+
+```
+data "aws_iam_policy_document" "quickwit_node" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "sqs:ReceiveMessage",
+      "sqs:DeleteMessage",
+      "sqs:ChangeMessageVisibility",
+      "sqs:GetQueueAttributes",
+    ]
+    resources = [aws_sqs_queue.s3_events.arn]
+  }
+  statement {
+    effect    = "Allow"
+    actions   = ["s3:GetObject"]
+    resources = ["${aws_s3_bucket.file_source.arn}/*"]
+  }
+}
+```
+
+Create the IAM user and credentials that will be used to
+associate this policy to your local Quickwit instance:
+
+```
+resource "aws_iam_user" "quickwit_node" {
+  name = "quickwit-filesource-tutorial"
+  path = "/system/"
+}
+
+resource "aws_iam_user_policy" "quickwit_node" {
+  name   = "quickwit-filesource-tutorial"
+  user   = aws_iam_user.quickwit_node.name
+  policy = data.aws_iam_policy_document.quickwit_node.json
+}
+
+resource "aws_iam_access_key" "quickwit_node" {
+  user = aws_iam_user.quickwit_node.name
+}
+```
+
+
+:::warning
+
+We don't recommend using IAM user credentials for running Quickwit nodes in
+production. This is just a simplified setup for the sake of the tutorial. When
+running on EC2/ECS, attach the policy document to an IAM roles instead.
+
+:::
+
+Download the [complete terraform script](../assets/sqs-file-source.tf) and
+deploy it using `terraform init` and `terraform apply`. After a successful
+execution, the outputs required to configure Quickwit will be listed. You can
+display the values of the sensitive outputs (key id and secret key) with:
+
+
+```bash
+terraform output quickwit_node_access_key_id
+terraform output quickwit_node_secret_access_key
+```
+
+## Run Quickwit
+
+[Install Quickwit locally](/docs/get-started/installation), then in your install
+directory, run Quickwit with the necessary access rights by replacing the
+`<quickwit_node_access_key_id>` and `<quickwit_node_secret_access_key>` with the
+matching Terraform output values:
+
+```bash
+AWS_ACCESS_KEY_ID=<quickwit_node_access_key_id> \
+AWS_SECRET_ACCESS_KEY=<quickwit_node_secret_access_key> \
+AWS_REGION=us-east-1 \
+./quickwit run
+```
+
+## Configure the index and the source
+
+In another terminal, in the Quickwit install directory, create an index:
+
+```bash
+cat << EOF > tutorial-sqs-file-index.yaml
+version: 0.7
+index_id: tutorial-sqs-file
+doc_mapping:
+  mode: dynamic
+indexing_settings:
+  commit_timeout_secs: 5
+EOF
+
+./quickwit index create --index-config tutorial-sqs-file-index.yaml
+```
+
+Replacing `<notification_queue_url>` with the corresponding Terraform output
+value, create a file source for that index:
+
+```bash
+cat << EOF > tutorial-sqs-file-source.yaml
+version: 0.8
+source_id: sqs-filesource
+source_type: file
+num_pipelines: 2
```

Comment:
It is worth adding a sentence or two explaining the point/effect of num_pipelines